Taken from, not defined in (#4718)
quicwg/base-drafts: draft-ietf-quic-invariants.md

# Notational Conventions

> Packet diagrams in this document use a format taken from {{QUIC-TRANSPORT}} to
> illustrate the order and size of fields.  That format is defined in this
> section.

Reviewer: ```suggestion
The format used by packet diagrams in this document is defined in this
section, and is the same as that used in {{QUIC-TRANSPORT}}.
```

Reviewer 2: ```suggestion
The format of packets is described using the notation defined in this section.
This notation is the same as that used in {{QUIC-TRANSPORT}}.
```

Reviewer: @martinthomson : your suggestion works for me.
